go-client/session/meta_call.go

```diff
@@ -141,7 +141,7 @@ func (c *metaCall) issueSingleMeta(ctx context.Context, curLeader int) bool {
 			c.lock.Lock()
 			c.metaIPAddrs = append(c.metaIPAddrs, addr)
 			c.metas = append(c.metas, &metaSession{
-				NodeSession: newNodeSession(addr, NodeTypeMeta),
+				NodeSession: newNodeSession(addr, NodeTypeMeta, false),
```

Comment:
The bool type true/false parameter is meaningless, how about using a `const` to define enable or disable metrics?